src/middlewared/middlewared/plugins/virt/stats.py

```diff
@@ -1,25 +1,22 @@
 import time
 
 from middlewared.event import EventSource
-from middlewared.schema import Dict, Str
+from middlewared.schema import Dict, Int
 from middlewared.plugins.reporting.realtime_reporting.cgroup import get_cgroup_stats
+from middlewared.validators import Range
 
 
 class VirtInstancesMetricsEventSource(EventSource):
```

Comment:
Why do we have this as its own separate event source instead of adding the logic to reporting.realtime event source?